serverless/verify-backup/src/server.ts

```diff
@@ -1,14 +1,33 @@
 import express, { Request, Response } from 'express'
 import bodyParser from 'body-parser'
+import { spawn } from 'child_process'
 
 const app = express()
 const PORT = process.env.PORT || 8080
 
 app.use(bodyParser.json())
 
-app.post('/', (req: Request, res: Response) => {
-  console.log('Incoming pubsub message', JSON.stringify(req.body))
-  res.sendStatus(200)
+app.post('/', async (req: Request, res: Response) => {
+  // set request timeout to prevent server from aborting
+  // before verify-backup process is complete
+  req.setTimeout(20 * 60 * 1000) // 20 mins
+
+  // run verify-backup script
+  const verifyBackup = await spawn('sh', ['scripts/verify-backup.sh'])
+  verifyBackup.stdout.on('data', (data) => {
+    console.log(data.toString())
```

Comment:
```suggestion
    console.log(data.toString().trim())
```

Minor nitpick, might want to add `trim` to prevent additional empty lines in the logs.